dvc/command/config.py

```diff
@@ -90,9 +108,22 @@ def _check(self, conf, remote, section, opt=None):
             )
 
     @staticmethod
-    def _format_config(config):
+    def _format_config(config, prefix=""):
         for key, value in flatten(config).items():
-            yield f"{key}={value}"
+            yield f"{prefix}{key}={value}"
+
+    @staticmethod
+    def _config_file_prefix(show_origin, config, level):
+        if not show_origin:
+            return ""
+
+        fname = config.files[level]
+
+        if level in ["local", "repo"]:
+            common = os.path.commonprefix([fname, Repo.find_root()])
+            fname = fname[len(common) + 1 :]
```

Comment:
Maybe just relpath? Or am I missing something?

Reply:
Yes, that is certainly more clear. Done.